client/components/PubEdge/PubEdge.js

```diff
@@ -109,41 +109,52 @@ const PubEdge = (props) => {
 		onKeyDown: handleClick,
 		role: 'link',
 		tabIndex: '0',
+		alt: title,
 	};
 
 	const maybeLink = (element, restProps = {}) => {
 		if (actsLikeLink) {
-			return element;
+			return (
+				<span {...restProps} className={classNames(restProps.className, 'link')}>
+					{element}
+				</span>
+			);
 		}
-		return (
-			<a href={url} {...restProps}>
-				{element}
-			</a>
-		);
+
+		return <a href={url}>{element}</a>;
+	};
+
+	const maybeWrapWithLink = (element, restProps = {}) => {
+		if (actsLikeLink) {
+			return (
+				<a href={url} {...restProps}>
+					{element}
+				</a>
+			);
+		}
+
+		return <div {...restProps}>{element}</div>;
```

Comment:
This function needs a better name. It just wraps the element with an anchor tag if `actsLinkLike` is true.

Reply:
Seems fine to me!